### PR TITLE
feat: VC Wallet Query By PresentationExchange

### LINKED ISSUE #2712: VC Wallet - GetAll
Get all contents by content type

### LINKED ISSUE #2713: VC Wallet - Query By PresentationExchange

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- PresentationExchange query support
- GetAll by content type support
- Closes #2712
- Closes #2713

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

File: pkg/wallet/query.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+		return nil, err
+	}
+
+	if len(vcs) == 0 {
+		return nil, ErrQueryNoResultFound
+	}
+
+	switch q {
+	case QueryByExample:
+		return nil, fmt.Errorf("to be implemented")
```
Comment (troyronda) [inline pkg/wallet/query.go:51]: ?

File: pkg/wallet/query.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	case QueryByFrame:
+		return nil, fmt.Errorf("to be implemented")
```
Comment (troyronda) [inline pkg/wallet/query.go:53]: ?

**Review by troyronda — APPROVED** [review]

File: pkg/wallet/query.go (hunk 1 above)
Comment (sudeshrshetty) [reply pkg/wallet/query.go:51]: Will be part of next PR

File: pkg/wallet/query.go (hunk 2 above)
Comment (sudeshrshetty) [reply pkg/wallet/query.go:53]: Will be part of next PR

**Pull request merged**